hawkular-inventory-bus/src/main/java/org/hawkular/inventory/bus/BusIntegration.java

```diff
@@ -61,21 +56,12 @@ public void configure(Configuration configuration) {
         this.configuration = configuration;
     }
 
-    public void start() throws NamingException, JMSException {
+    public void start() {
         if (namingContext != null) {
```

Comment:
Also the `stop()` method will throw an NPE on trying to close the `namingContext`.